Automattic/jetpack
# Implemented a CSS nudge section for the Customizer section.

Fixes https://github.com/Automattic/wp-calypso/issues/51589

#### Changes proposed in this Pull Request:

* A implementation for Atomic and WPCOM for Additional CSS nudge in Customizer

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






Atomic:
- Setup your local JP site to mimic an Atomic env for Automattic/wpcomsh#quick-start or use A8C WP env.
- Checkout 693-gh-wpcomsh branch on your wpcomsh/jetpack environment.
- Make sure that your site is on a free plan (you can hardcode the Atomic_Plan_Manager::current_plan_slug to return 'free' or 'business').
- On a free site, go to Appearance > Customize > Additional CSS.
- Make sure that the nudge is displayed instead of the actual "Additional CSS" feature.
- On a business Atomic site, please go to Appearance > Customize > Additional CSS.
- Make sure that the "Additional CSS" section still loads correctly. 

Simple:
- Apply the patch D62023-code on your sandbox.
- Go to a free site and click on Appearance > Custom CSS.
- Make sure that the Customizer is loaded and the nudge message is loaded instead of the actual "Additional CSS" section.
- Go to a premium site and click on Appearance > Custom CSS and make sure that the actual "Additional CSS" section is loaded.

Since the implementation now resides in Jetpack the WPCOM patch was manually updated to contain the Jetpack changes, therefore, D62023-code should also be reviewed.
